SalesforcePy/__init__.py

```diff
@@ -1,7 +1,18 @@
-from __future__ import absolute_import
-from . import sfdc
+import warnings
 
-name = 'SalesforcePy'
-client = sfdc.client
+# the following lines are intended to provide backwards compatibility and should be removed once the SalesforcePy
+# package has been removed
+import salesforce
+from salesforce import name, client, LoginException  # noqa: F401
 
-LoginException = sfdc.LoginException
+sfdc = salesforce.sfdc
+chatter = salesforce.chatter
+commons = salesforce.commons
+jobs = salesforce.jobs
+wave = salesforce.wave
+
+warnings.warn(
```

Comment:
@demianbrecht I think this looks good, but before I approve, can you help me understand your proposal for the next major release here? Is the idea to rename the package entirely to `salesforce`?